From 9346fe8456a56fe2c4272b92d61e957ae59dfbfb Mon Sep 17 00:00:00 2001
From: reaper <theunbounddeveloper@outlook.com>
Date: Thu, 6 Aug 2026 09:33:27 -0400
Subject: [PATCH] v2.2.0: version bump (first-boot loop fix + Gatekeeper
 LAN-DNS carve-out + WiFiPro preamble lock + USB Disks tab + de-cloud
 console-noise batch)

---
 release/src-rt/version.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/release/src-rt/version.conf b/release/src-rt/version.conf
index 2765c923c5..c671792802 100644
--- a/release/src-rt/version.conf
+++ b/release/src-rt/version.conf
@@ -4,6 +4,6 @@ KERNEL_VER_AM=$(subst .,,$(KERNEL_VER))
 FS_VER_AM=$(subst .,,$(FS_VER))
 
 SERIALNO=102.8
-EXTENDNO=Reaper_v2.1.9
+EXTENDNO=Reaper_v2.2.0
 
 RCNO=0
-- 
2.25.1

